scarb-metadata/tests/command.rs

```diff
@@ -70,6 +70,7 @@ fn manifest_path() {
 fn init_project(t: &TempDir) {
     Command::new(scarb_bin())
         .args(["init", "--name", "hello"])
+        .env("SCARB_TEST_RUNNER", "cairo-test")
```

Comment:
why is this line needed everywhere? can this set of changes be avoided?

Reply:
This line is repeated twice in the functions that are the same. If you mean `args(CAIRO_NATIVE_RUNNER)`, this can be avoided if you want to be less explicit in tests. We can set it in the `pub fn std(self) -> StdCommand` function.

Reply:
If it's not needed for these tests, then I'd stick with default behaviour because this is out of interest of the test